docs/_posts/examples/3400-01-04-geojson-layer-in-stack.html

```diff
@@ -16,7 +16,12 @@
 });
 
 map.on('load', function () {
-
+    // Insert the layer beneath any symbol layer.
+    var layers = map.getStyle().layers.reverse();
+    var labelLayerIdx = layers.findIndex(function (layer) {
```

Comment:
I think we could make this example easier to understand by elaborating a bit in the comment i.e "find the index of the bottom-most `type: symbol` layer in the map style"

Reply:
@mollymerp sounds good. I'll take a look when I have some time. thanks for review 👍